packages/plugins/rpc-manager/src/lib/eth_sendTransaction.ts

```diff
@@ -1,18 +1,23 @@
 import async from "async";
-import { Callback, Embark, EmbarkEvents } from "embark-core";
+import { Callback, Embark, EmbarkEvents, EmbarkPlugins } from "embark-core";
 import { __ } from "embark-i18n";
 import Web3 from "web3";
 const { blockchain: blockchainConstants } = require("embark-core/constants");
 import RpcModifier from "./rpcModifier";
+import cloneDeep from "lodash.clonedeep";
 
 export default class EthSendTransaction extends RpcModifier {
   private signTransactionQueue: any;
   private nonceCache: any = {};
+  private plugins: EmbarkPlugins;
   constructor(embark: Embark, rpcModifierEvents: EmbarkEvents, public nodeAccounts: string[], public accounts: any[], protected web3: Web3) {
     super(embark, rpcModifierEvents, nodeAccounts, accounts, web3);
 
+    this.plugins = embark.config.plugins;
+
     embark.registerActionForEvent("blockchain:proxy:request", this.ethSendTransactionRequest.bind(this));
 
+    // TODO: pull this out in to rpc-manager/utils once https://github.com/embarklabs/embark/pull/2150 is merged.
```

Comment:
This could be done now, but it would be easier to do in another PR

Reply:
Yes, planning to do that in another PR.